packages/bpk-component-input/src/BpkInput.tsx

```diff
@@ -70,7 +64,7 @@ class BpkInput extends Component<Props, State> {
     } = this.props;
 
     // Used as a ref for focussing the input when cleared.
-    let ref = null;
+    let ref: HTMLElement | null = null;
```

Comment:
Should this be HTMLInputElement?

Reply:
Which is why you then use the type assertion within there 👍

Reply:
I was completely wrong, it's not actually used for both 😅 changed it to HTMLInputElement!